mintme/src/graphql/backendClient.ts

```diff
@@ -43,6 +44,9 @@ export async function backendClient<T extends Record<string, unknown>, X = Objec
     return data
   } catch (e) {
     if (e & e.response) {
+      if (e?.response?.status === 401) {
```

Comment:
Is this supposed to be 401 or 403?

Reply:
401 is thrown by our API route and backendClient only calls our api route so its correct

Reply:
Sounds good